sphinx/source/models.rst

```diff
@@ -165,6 +165,11 @@ Model Conversion
 .. automodule:: botorch.models.converter
     :members:
 
+Inducing Point Allocators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. automodule:: botorch.models.utils.InducingPointAllocator
```

Comment:
This needs to be the module name not the class name
```suggestion
.. automodule:: botorch.models.utils.inducing_point_allocators
```

Reply:
sorry, stupid me